### ISSUE TITLE
> **I tryed styling the Mui dataTable like you suggested and it still isn't working**

### ISSUE THREAD

**Issue by matsuu2tatsuya** [gregnb/mui-datatables#1878]
> **I tryed styling the Mui dataTable like you suggested and it still isn't working**
> 
> Here is my code
> 
> import MUIDataTable from "mui-datatables"; import { createTheme, ThemeProvider } from "@mui/material/styles";
> 
> function Customers() { const columns = ["Name", "Company", "City", "State"];
> 
> const data = [ ["Joe James", "Test Corp", "Yonkers", "NY"], ["John Walsh", "Test Corp", "Hartford", "CT"], ["Bob Herm", "Test Corp", "Tampa", "FL"], ["James Houston", "Test Corp", "Dallas", "TX"], ["James Houston", "Test Corp", "Dallas", "TX"], ];
> 
> const options = { filterType: "multiselect", responsive: "scroll", };
> 
> return ( <ThemeProvider theme={createTheme({ components: { MUIDataTableBodyCell: { styleOverrides: { root: { backgroundColor: "#FF0000", }, }, }, }, })} > <MUIDataTable sx={{ height: "100px" }} title={"Customers"} data={data} columns={columns} options={options} /> ); }
> 
> export default Customers;

I'm running into a similar problem.

It appears that I can override the Style of an existing mui Component, but the Style is not reflected in the mui-datatables.

[ style overrides ]
ok => MuiTableRow, MuiTableCell, ...etc (mui official className)
failed => MUIDataTableHeadCell, MUIDataTableHeadRow, ...etc (mui-datatable className)

styleOverrides
It was working fine until February 19.

_Originally posted by @matsuu2tatsuya in https://github.com/gregnb/mui-datatables/issues/1842#issuecomment-1047418962_

**Comment by kkaplita** [comment]
Any solutions?

**Comment by kachmul2004** [comment]
Seems I also not able to style this MUI-datables using mui 5.5.0 themes.

**Comment by StefanIgnjatovic12** [comment]
Anyone else still having this issue? Tried the override as in #162 but it's not working

`export const getMuiTheme = () =>
    createTheme({
        overrides: {
            MUIDataTableHeadCell: {
                fixedHeaderCommon: {
                    backgroundColor: "blue"
                }
            }
        }
    });`

I'm actually just trying to make the column label bold but I can't get any changes to apply

**Comment by kachmul2004** [comment]
@StefanIgnjatovic12 just downgrade to 4.0.0. That's the only thing that seemed to work for me

**Comment by garronej** [comment]
It's fixed in the PR #1882.  It just needs to get merged.

**Comment by garronej** [comment]
Seems like this project is no longer actively maintained...  
A workaround that I can offer (to yarn users) is to pin `tss-react` to the newer version in your `package.json` like so:  

```json
"resolutions": {
    "tss-react": "3.6.0"
},
"dependencies": {
    "tss-react": "^3.6.0"
}
```

Once the PR get merged you will be able to remove `tss-react` from your dependencies.
